extension/content/personas.css

```diff
@@ -90,6 +95,20 @@
   font-weight: bold !important;
 }
 
+#personas-toolbar-button.seamonkey, #personas-toolbar-button.seamonkey:hover, #personas-toolbar-button.seamonkey:hover:active {
+  list-style-image: url("chrome://personas/content/personas_30x30.png");
+  -moz-image-region: rect(0, 30px, 30px, 0px);  
+}
+
+toolbar[iconsize="small"] #personas-toolbar-button.seamonkey {
+  list-style-image: url("chrome://personas/content/personas_20x20.png");
```

Comment:
But line 100 is required because **hover** and **active** pseudos change the -moz-image-region. To show the icon properly, -moz-image-region is set to 30px 30px as the image dimension.